packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -14,9 +14,12 @@ import { ProjectStructure } from "../../../common/src/project/structure.js";
  * generateTestData will run yext sites generate-test-data and return true in
  * the event of a successful run and false in the event of a failure.
  */
-export const generateTestData = async (): Promise<boolean> => {
+export const generateTestData = async (domain?: string): Promise<boolean> => {
   const command = "yext";
-  const args = ["sites", "generate-test-data"];
+  let args = ["sites", "generate-test-data"];
+  if (domain) {
+    args = args.concat("--hostname", domain);
+  }
 
   async function generate() {
     const childProcess = spawn(command, args);
```

Comment:
nit and clearly this was here before, but below I think it can just return `!!exitCode`. Also instead of 
```ts
return new Promise((resolve) => {
    resolve(generate());
  });
```
I think just  `return generate()` should be fine?